<launch>

     <node pkg="landing_module" name="landing_module_node" type="landing_module_node" output="screen">
        <remap from="landing_module_node/image" to="/cgo3_camera/image_raw"/>
        <remap from="landing_module_node/pose" to="/mavros/local_position/pose"/>
        <param name="h_fov" type="double" value="90"/>
        <param name="search_altitude" type="double" value="50"/>
    </node>
</launch>
